configureBuild.sh

```diff
@@ -127,10 +127,22 @@ setVariablesForConfigure() {
     [ "$openjdk_core_version" == "${JDKHEAD_CORE_VERSION}" ]; then
     local jdk_path="jdk"
     local jre_path="jre"
+    case "${BUILD_CONFIG[OS_KERNEL_NAME]}" in
+    "darwin")
+      local jdk_path="jdk-bundle/jdk-*.jdk"
```

Comment:
I assume `local` in bash is local to the scope it's in (i.e. the if statement)

Reply:
yup